web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/formidling/beregningsgrunnlag/BeregningsgrunnlagFormidlingV2DtoTjeneste.java

```diff
@@ -120,6 +120,16 @@ private boolean erSeksAvDeTiBeste() {
         return besteBeregnetBeløp.compareTo(ordinærtBeregnetBeløp) > 0;
     }
 
+    private BigDecimal hentOverstyrtEllerBeregnetHvisFinnes(BeregningsgrunnlagPrStatusOgAndel andel) {
+        if (andel.getOverstyrtPrÅr() != null ) {
```

Comment:
Funksjonell stil for samme kode :-) 
return Optional.ofNullable(andel.getOverstyrtPrÅr())
       .or(() -> Optional.ofNullable(andel.getBeregnetPrÅr())
       .orElse(BigDecimal.ZERO)

Reply:
Oppdatert :-)